lib/util.js

```diff
@@ -573,6 +628,8 @@ module.exports = {
     download_youtube_thumbnail,
     encode_uri,
     fetch_html,
+    fetch_news_article,
+    fetch_blog_article,
```

Comment:
Alphabetical order

Reply:
This is also not being checked in our lint rules. Do you think we should enable sort-keys? https://eslint.org/docs/rules/sort-keys

Reply:
Yes, but see above.